components/form-toggle/style.scss

```diff
@@ -52,11 +38,35 @@ $toggle-border-width: 2px;
 		border-radius: 50%;
 		background: $dark-gray-500;
 		transition: 0.1s transform ease;
+	}
 
-
-		.components-form-toggle.is-checked & {
+	&.is-checked {
+		&:after {
 			background-color: $white;
 			transform: translateX( $toggle-width - ( $toggle-border-width * 4 ) - ( $toggle-height - ( $toggle-border-width * 4 ) ) );
 		}
+
+		&:before {
+			background-color: $blue-medium;
+			border: $toggle-border-width solid $blue-medium;
+		}
 	}
 }
+
+.components-form-toggle__input[type=checkbox] {
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	opacity: 0;
+	margin: 0;
+	padding: 0;
+	z-index: z-index( '.components-form-toggle__input' );
```

Comment:
I'm wondering if some logical groupings in the `_z-index.scss`, even just a comment, could help avoid this temptation of assigning it a value _a little more than the next highest_. Like "Relative Siblings", "Layout", "Dialogs", etc. We have some grouping already, but mostly by related components. Dunno if that's a more sustainable pattern than lower-to-highest. It's a difficult problem, and one which I don't think needs to be solved here, but I expect will continue to surface over time.

Related discussion at #637